content/en/security/threats/investigate_agent_events.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Investigate Agent Events
+kind: documentation
+disable_toc: false
+further_reading:
+  - link: "/security/default_rules/?category=cat-csm-threats#all"
+    tag: "Documentation"
+    text: "Explore CSM Threats detection rules"
+  - link: "/security/threats/workload_security_rules"
+    tag: "Documentation"
+    text: "Learn how to manage CSM Threats detection rules"
+  - link: "/security/notifications/"
+    tag: "Documentation"
+    text: "Learn more about security notifications"
+  - link: "https://www.datadoghq.com/blog/datadog-csm-windows/"
+    tag: "Blog"
+    text: "Secure your Windows workloads with Datadog Cloud Security Management"
+---
+
+
+This topic explains how to use the Agent Events explorer to query and review the Datadog Agent threat detection events generated by the [out-of-the-box (OOTB) detection rules][12].
+
+The Datadog Agent evaluates system activity on the Agent host. When activity matches an Agent rule expression, the Agent generates a detection event and passes it to the Datadog backend.
+
+If an event matches an Agent detection rule *and* a backend Threat detection rule, a signal is created and displayed in [Signals][11] (`Agent detection rule + backend Threat detection rule = Signal`).
+
+With the [Agent Events explorer][13], you can investigate Agent Events separately from signals. You can review the host path where the event happened, and view the event's attributes, metrics, and processes. You can also review the Agent rule that generated the event and view triage and response instructions.
+
+## Proactively block threats with Active Protection
+
+By default, all OOTB Agent threat detection rules are enabled and actively monitoring for threats.
+
+[Active Protection][12] enables you to proactively block and terminate threats identified by the Datadog Agent threat detection rules.
```

Comment:
Did you intend to link this to the OOTB Rules page? Or should the link point to the [Active Protection article](https://docs.datadoghq.com/security/cloud_security_management/guide/active-protection/) instead?

Reply:
fixed it. thanks. great catch.